lms/djangoapps/course_api/tests/test_api.py

```diff
@@ -75,8 +84,8 @@ def setUpClass(cls):
         cls.staff_user = cls.create_user('staff', is_staff=True)
 
     def test_get_existing_course(self):
-        result = self._make_api_call(self.honor_user, self.honor_user.username, self.course.id)
-        self.assertEqual(self.expected_course_data, result)
+        course = self._make_api_call(self.honor_user, self.honor_user.username, self.course.id)
```

Comment:
while not exactly changed in this PR, the two `_make_api_call` helper methods in this module have a subtle difference which was initially confusing upon review.  (specifically, the way the one used here is really expecting a username but seems to expect a User object, while other one actually expects a User object and extracts the username itself.)

so, optional, but seems like it would be beneficial and fairly straightforward to clean that up in this PR.